### ISSUE TITLE
Intent to implement: Add radio buttons type to knobs

### ISSUE THREAD

**Issue by nfiniteset** [storybooks/storybook#3872]
### Intent to implement

@brycemhammond and I want to add radio buttons knob type. Select has the data model covered, but it would help usability in some of our stories to have all the options visible at once. 

I've built a proof of concept [on a branch](https://github.com/nfiniteset/storybook/tree/add-radio-buttons). Is the core team interested in accepting this? If so we'll carry the work farther wrt testing, documentation, etc.

### POC is built against:

- @storybook/react 3.4.8
- @storybook/addon-knobs 3.4.8

### Screenshot

![storybook-radio-buttons](https://user-images.githubusercontent.com/178653/42588089-9f3f480a-84f1-11e8-8071-85351fb399c7.gif)

### Code Snippets

The API is identical to the Select type

```js
// within a story definition:
    const fruits = {
      apple: 'Apple',
      banana: 'Banana',
      cherry: 'Cherry',
    };
    const fruit = select('Fruit', fruits, 'apple');

    const otherFruits = {
      Lime: 'lime',
      Coconut: 'coconut',
      Tomato: 'tomato',
    };
    const otherFruit = radioButtons('Other Fruit', otherFruits, 'lime');
```

**Comment by igor-dv** [comment]
@nfiniteset, looks like a good addition to the knobs addon. You are welcome to PR it, but of course against the latest master 👍

**Comment by Frikki** [comment]
Much appreciated addition ❤️

**Comment by Hypnosphi** [comment]
Released as (`4.0.0-alpha.15`)[https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.15]
